pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -7,12 +7,18 @@ SPDX-License-Identifier: Apache-2.0
 package didexchange
 
 import (
+	"encoding/json"
```

Comment:
the imports should be grouped as:

- stdlib
- third party
- aries-framework-go

Reply:
done